README.md

```diff
@@ -176,14 +176,11 @@ config.action_mailer.delivery_method = :ses # or :sesv2
 config.action_mailbox.ingress = :ses
 ```
 
-2. Configure which _SNS_ topics will be accepted and what region the emails will be stored in when using S3 (plus any other desired options):
+2. Configure which _SNS_ topic will be accepted and what region the emails will be stored in when using S3 (plus any other desired options):
```

Comment:
Feel free to disregard, but it may be worth mentioning that the `s3_client_options[:region]` does not need to match the region of the subscribed_topic, but must match the region for the configured s3 bucket.  (I find this allowed mismatch confusing personally....)